contrib/agent/build.gradle

```diff
@@ -93,6 +95,8 @@ shadowJar {
   // classes, which are loaded by the same classloader (the system classloader).
   // Byte Buddy:
   relocate 'net.bytebuddy', agentRepackaged + '.bytebuddy'
+  // Config:
+  relocate 'com.typesafe.config', agentRepackaged + '.config'
```

Comment:
not quite sure the impact of source 1.6, but target 1.8.. suppose the JRE 6 tests will ensure this works with JRE 6? Or will we have to watch execution paths? https://github.com/lightbend/config/blob/master/config/build.sbt#L75

Reply:
Nice catch! I haven't looked into that project's build scripts before.

According to https://github.com/lightbend/config/blob/master/README.md#binary-releases, the last version supporting Java 6 is 1.2.1, which I'm using in this PR. It's interesting to see that typesafe config is using source 1.6 and target 1.8 (https://github.com/lightbend/config/commit/e72decad9251ea2c31e7e53dbfc9902f0f922194) - I haven't looked that close. Ideally, I'd like to use a newer version of typesafe config, but not bothering about that for now.

In any case the integration tests running with JRE 6 will catch any obvious issue (class file version, usage of java 7+ APIs at runtime).